connect/src/com/telenor/connect/utils/ConnectUtils.java

```diff
@@ -25,12 +25,16 @@ public static void parseAuthCode(String callbackUrl, ConnectCallback callback) {
         Validator.notNullOrEmpty(callbackUrl, "callbackUrl");
 
         Uri uri = Uri.parse(callbackUrl);
-        Validator.validateAuthenticationState(uri.getQueryParameter("state"));
+        if (!Validator.validState(uri.getQueryParameter("state"))) {
```

Comment:
Nothing ;).
I guess you will want to delete validateAuthenticationState now as it's not used any more.